Timeout during installation:  assets:install web --symlink --relative --ansi" exceeded the timeout of 60 seconds.

# Bug Report
Install script crashes with the following error:
```
 [ERROR] The following errors were encountered during installation
 * The process "C:\xampp\php\php.exe "C:\src\cispa\pimcore2/bin/console" assets:install web --symlink --relative --ansi" exceeded the timeout of 60 seconds.
```

### Expected behavior
Installer should run through.

### Actual behavior
See above.

### Steps to reproduce
Install pimcore skeleton via composer; run the install script.


This issue occured for some of our project team, not for others. I am suspecting it has to do with our environment (Win 10, using git bash), but I am unsure why it occurs. Interestingly, installing the assets separately works, BUT it installs them as copies even thogh the --symlink option is given. Maybe this is the reason why it takes so long.
```
$ bin/console assets:install web --symlink --relative --ansi

 Trying to install assets as relative symbolic links.

 --------- -------------------- ----------------
            Bundle               Method / Error
 --------- -------------------- ----------------
  WARNING   PimcoreCoreBundle    copy
  WARNING   PimcoreAdminBundle   copy
 --------- -------------------- ----------------

 ! [NOTE] Some assets were installed via copy. If you make changes to these assets you have to run this command again.


 [OK] All assets were successfully installed.
```

--- a/lib/Tool/AssetsInstaller.php
+++ b/lib/Tool/AssetsInstaller.php
@@ -49,6 +49,7 @@ class AssetsInstaller
     public function install(array $options = []): Process
     {
         $process = $this->buildProcess($options);
+        $process->setTimeout(240);
         $process->run($this->runCallback);
 
         if (!$process->isSuccessful()) {

